packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -76,6 +76,10 @@ export default function EditPostPreferencesModal() {
 		closeGeneralSidebar();
 	};
 
+	const colorContrastCheckAAA = useSelect( ( select ) => {
+		const { isFeatureActive } = select( editPostStore );
+		return isFeatureActive( 'strictColorContrastChecks' );
+	}, [] );
```

Comment:
This can be selected from the preferences store instead of using `isFeatureActive` (which I think is deprecated, or should be deprecated).